src/test/java/teammates/test/cases/datatransfer/FeedbackMcqQuestionDetailsTest.java

```diff
@@ -385,6 +385,33 @@ public void testValidateQuestionDetails_negativeOtherWeight_errorsReturned() {
         assertEquals(Const.FeedbackQuestion.MCQ_ERROR_INVALID_WEIGHT, errors.get(0));
     }
 
+    @Test
+    public void testValidateQuestionDetails_duplicateMcqOptions_errorReturned() {
```

Comment:
BTW, you should test directly the method `validateQuestionDetails`.

`extractQuestionDetails` is deprecated in V7 now.

Reply:
done